packages/eslint-plugin-pf-codemods/src/rules/helpers/getDeclarationString.ts

```diff
@@ -0,0 +1,8 @@
+import { ImportDefaultSpecifierWithParent } from "./interfaces";
+
+/** Gets the import path string for a default import */
+export function getDeclarationString(
```

Comment:
More of a nit so not a blocker, but would `getDefaultDeclarationString` or something similar be more accurate?